### PR TITLE
feat: v10 beta

### PR DESCRIPTION
**Pull request by juliusmarminge**
Closes #<issue>

## ✅ Checklist

- [ ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

bumps trpc v10 to beta

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Review by nexxeln — APPROVED** [review]

**Comment by nexxeln** [comment]
CI is failing though. Are there any breaking changes?

**Comment by juliusmarminge** [comment]
> CI is failing though. Are there any breaking changes?

The npm package isn't published yet, will be sometime during the day :)

**Comment by nexxeln** [comment]
> > CI is failing though. Are there any breaking changes?
> 
> The npm package isn't published yet, will be sometime during the day :)

Ah makes sense

File: cli/src/installers/index.ts [hunk 1]
```diff
@@ -35,14 +35,10 @@ export const dependencyVersionMap = {
...
   // tRPC
-  "@trpc/client": "10.0.0-proxy-alpha.81",
-  "@trpc/server": "10.0.0-proxy-alpha.81",
-  "@trpc/react": "10.0.0-proxy-alpha.81",
-  "@trpc/next": "10.0.0-proxy-alpha.81",
-  // "@trpc/client": "10.0.0-beta.1",
-  // "@trpc/server": "10.0.0-beta.1",
-  // "@trpc/react": "10.0.0-beta.1",
-  // "@trpc/next": "10.0.0-beta.1",
+  "@trpc/client": "10.0.0-beta.1",
```
Comment (juliusmarminge) [inline cli/src/installers/index.ts:38]:
```suggestion
  "@trpc/client": "10.0.0-beta.0",
```

File: cli/src/installers/index.ts [hunk 2]
```diff
@@ -35,14 +35,10 @@ export const dependencyVersionMap = {
...
+  "@trpc/server": "10.0.0-beta.1",
```
Comment (juliusmarminge) [inline cli/src/installers/index.ts:39]:
```suggestion
  "@trpc/server": "10.0.0-beta.0",
```

File: cli/src/installers/index.ts [hunk 3]
```diff
@@ -35,14 +35,10 @@ export const dependencyVersionMap = {
...
+  "@trpc/react": "10.0.0-beta.1",
```
Comment (juliusmarminge) [inline cli/src/installers/index.ts:40]:
```suggestion
  "@trpc/react": "10.0.0-beta.0",
```

File: cli/src/installers/index.ts [hunk 4]
```diff
@@ -35,14 +35,10 @@ export const dependencyVersionMap = {
...
+  "@trpc/next": "10.0.0-beta.1",
```
Comment (juliusmarminge) [inline cli/src/installers/index.ts:41]:
```suggestion
  "@trpc/next": "10.0.0-beta.0",
```

File: cli/src/installers/index.ts [hunk 5]
```diff
@@ -35,14 +35,10 @@ export const dependencyVersionMap = {
...
   // tRPC
-  "@trpc/client": "10.0.0-proxy-alpha.81",
-  "@trpc/server": "10.0.0-proxy-alpha.81",
-  "@trpc/react": "10.0.0-proxy-alpha.81",
-  "@trpc/next": "10.0.0-proxy-alpha.81",
-  // "@trpc/client": "10.0.0-beta.1",
-  // "@trpc/server": "10.0.0-beta.1",
-  // "@trpc/react": "10.0.0-beta.1",
-  // "@trpc/next": "10.0.0-beta.1",
+  "@trpc/client": "10.0.0-beta.0",
```
Comment (juliusmarminge) [inline cli/src/installers/index.ts:38]:
```suggestion
  "@trpc/client": "10.0.0-proxy-beta.0",
```

File: cli/src/installers/index.ts [hunk 6]
```diff
@@ -35,14 +35,10 @@ export const dependencyVersionMap = {
...
+  "@trpc/server": "10.0.0-beta.0",
```
Comment (juliusmarminge) [inline cli/src/installers/index.ts:39]:
```suggestion
  "@trpc/server": "10.0.0-proxy-beta.0",
```

File: cli/src/installers/index.ts [hunk 7]
```diff
@@ -35,14 +35,10 @@ export const dependencyVersionMap = {
...
+  "@trpc/react": "10.0.0-beta.0",
```
Comment (juliusmarminge) [inline cli/src/installers/index.ts:40]:
```suggestion
  "@trpc/react": "10.0.0-proxy-beta.0",
```

File: cli/src/installers/index.ts [hunk 8]
```diff
@@ -35,14 +35,10 @@ export const dependencyVersionMap = {
...
+  "@trpc/next": "10.0.0-beta.0",
```
Comment (juliusmarminge) [inline cli/src/installers/index.ts:41]:
```suggestion
  "@trpc/next": "10.0.0-proxy-beta.0",
```

**Pull request merged**